controllers/v1/observationsController.js

```diff
@@ -982,42 +993,67 @@ module.exports = class Observations extends Abstract {
                     result: {}
                 };
 
+                //eG change
                 let observationDocument = await database.models.observations.findOne({ 
                     _id: req.params._id, 
                      createdBy: req.userDetails.userId,
                      status: {$ne:"inactive"}, 
-                     entities: ObjectId(req.query.entityId) }).lean();
+                     entities: req.query.entityId }).lean();
 
                 if (!observationDocument) {
                     return resolve({ 
                         status: httpStatusCode.bad_request.status, 
                         message: messageConstants.apiResponses.OBSERVATION_NOT_FOUND
                     });
                 }
-               
-                let entityQueryObject = { _id: req.query.entityId, entityType: observationDocument.entityType };
-                let entityDocument = await database.models.entities.findOne(
-                    entityQueryObject,
-                    {
-                        metaInformation: 1,
-                        entityTypeId: 1,
-                        entityType: 1,
-                        registryDetails: 1
+               //eG code
+                let entityResult = [];
+                let filterData = {
+                    "id" : req.query.entityId
```

Comment:
@VISHNUDAS-tunerlabs why this filter is mssing 
entityType: observationDocument.entityType